build.sh

```diff
@@ -96,5 +159,15 @@ for theme_src_dir in "$SRC_DIR"/*; do
 	convert_to_x11cursor "$theme_build_dir" "$theme_out_dir"/cursors
 	create_aliases "$theme_out_dir"/cursors
 
+	create_hyprcursors "$theme_src_dir" "$theme_out_dir"
+	create_hyprcursor_aliases "$theme_out_dir"/hyprcursors
+
 	cp -f "$theme_src_dir/index.theme" "$theme_out_dir"/
+	theme_comment="$(grep Comment "$theme_src_dir"/index.theme)"
+	cat >"$theme_out_dir"/manifest.hl <<- EOM
+		name = $theme_name
+		description = ${theme_comment#Comment=}
+		version = $(git show -s --format=%cd --date=format:%Y%m%d HEAD)
+		cursor_directory = hyprcursors
```

Comment:
It should be `cursors_directory` instead of `cursor_directory`
https://github.com/hyprwm/hyprcursor/blob/main/docs/MAKING_THEMES.md#manifest

Reply:
So it should. I've fixed.